### PR TITLE
CoC fixes

### PR DESCRIPTION
**Pull request by patrickdupuis**

### CODE REVIEW

File: community/code-of-conduct.md [hunk 1]
```diff
@@ -57,10 +58,11 @@ Unacceptable behaviors include: intimidating, harassing, abusive, culturally app
 
 Harassment includes, but is not limited to:
 
-* Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, ethnicity, age, regional discrimination, political or religious affiliation
+* Harmful or prejudicial verbal or written comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, ethnicity, age, regional discrimination, political or religious affiliation
```
Comment (telephon) [inline community/code-of-conduct.md]: I think this is a real improvement, thank you. "Offensive" has become a really abused word and by now needs further qualification.

File: community/code-of-conduct.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ sort_order: 7
 
 ## Purpose
 
-A primary goal of our community, our user groups and mailing lists, symposia and other events is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all.
+A primary goal of our community, our user groups and mailing lists, symposia and other events is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
```
Comment (telephon) [inline community/code-of-conduct.md]: I'm not so sure about why we need this extra here: `for all` is absolutely definite, and any addition seems to give it a specificity that is not intended (at least I think so).

**Comment by brianlheim** [comment]
@patrickdupuis

 I actually didn't know this existed! How would you feel about making these changes instead to the code of conduct in the main SuperCollider repository, and then replacing this one with a link to that? I feel like it's not a great idea to have 2 of these floating around

File: community/code-of-conduct.md (hunk 1 above)
Comment (patrickdupuis) [reply community/code-of-conduct.md]: I agree. This change was taken from the Berlin Code of Conduct.

File: community/code-of-conduct.md (hunk 2 above)
Comment (patrickdupuis) [reply community/code-of-conduct.md]: This paragraph is taken from the Berlin CoC. I'm trying to make our text match theirs. I'd like to  keep this addition, but can remove it if that's preferred. @snappizz @brianlheim what do you think?

**Comment by patrickdupuis** [comment]
Absolutely, the repo version should be linked to from the website. I'll do this later today.

File: community/code-of-conduct.md (hunk 2 above)
Comment (snappizz) [reply community/code-of-conduct.md]: i support the addition.

**Comment by patrickdupuis** [comment]
I'll close this when supercollider/supercollider#3914 is merged. The website should link to the main repo CoC instead of containing its own duplicate copy.

**Comment by brianlheim** [comment]
Left comments on supercollider/supercollider#3914

**Pull request closed without merging**